tests/utilities/test_logger.py

```diff
@@ -172,3 +175,37 @@ def test_add_prefix():
     assert "prefix-metric2" not in metrics
     assert metrics["prefix2_prefix-metric1"] == 1
     assert metrics["prefix2_prefix-metric2"] == 2
+
+
+def test_name(tmpdir):
+    """Verify names of loggers are concatenated properly."""
+    logger1 = CSVLogger(tmpdir, name="foo")
+    logger2 = CSVLogger(tmpdir, name="bar")
+    logger3 = CSVLogger(tmpdir, name="foo")
+    logger4 = CSVLogger(tmpdir, name="baz")
+    loggers = [logger1, logger2, logger3, logger4]
+    name = _name([])
+    assert name == ""
+    name = _name([logger3])
+    assert name == "foo"
+    name = _name(loggers)
+    assert name == "foo_bar_baz"
+    name = _name(loggers, "-")
+    assert name == "foo-bar-baz"
+
+
+def test_version(tmpdir):
+    """Verify names of loggers are concatenated properly."""
```

Comment:
```suggestion
    """Verify versions of loggers are concatenated properly."""
```
this now looks like the most trivial unit test.. haha

Reply:
Trivial... but it does make you think about the edge cases :)